pset-functions.js

```diff
@@ -148,6 +196,18 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+function doubleLetters(word){
+  for(i = 0; i <= word.length; i++) {
+        for(var j = i+1; j <= word.length; j++) {
+            if(word[j] == word[i]) {
```

Comment:
I like that you took advantage the different loop's iterator to serve as the index for adjacent elements in the string. Be careful with nested loops however, it can really slow down your code. Instead, take advantage that you can use bracket notation and the original for loop's iterator; like in this [example](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L225).